src/Bokeh.jl

```diff
@@ -1,12 +1,15 @@
+VERSION >= v"0.4.0-dev+6641" && __precompile__()
```

Comment:
surely since the code now only supports `v0.4` and since that is released we can just remove the check.